keyper/__init__.py

```diff
@@ -12,12 +11,14 @@
 import subprocess
 import tempfile
 import uuid
+from distutils.version import StrictVersion
```

Comment:
I've never actually seen any documentation on whether `from`s should be inline or at the bottom. I've always done inline, alphabetical. The PEP8 document doesn't state either. I'd be curious to know if there's anything written on this anywhere? 

However, I don't mind either way, so I'm fine with this change.

Reply:
This was actually a very interesting comment so thanks @dalemyers !

I've been using PyCharm's [Optimize Imports](https://www.jetbrains.com/help/pycharm/creating-and-optimizing-imports.html) for a long time and just kind of assumed that `from ...` imports should come after `import ...` since that is how it automatically handles this. You are correct that PEP8 does not explicitly mention how to handle this particular case and that got me looking for more examples. I found a couple stackoverflow posts but the answer seems to be very subjective too.

One link is from an old reddit Python style guide which explicitly mentions the order of `import` before `from ... import`:

* https://github.com/reddit-archive/reddit/wiki/PythonImportGuidelines#ordering-of-import-statements

I created a test file to try out a couple different ideas:

```python
# hmm.py
from datetime import datetime
import string

# I had to use each import somehow or PyCharm's Optimize Imports
# would delete the unused imports too
d = datetime.now()
s = string.ascii_letters

```

[black](https://pypi.org/project/black/) doesn't seem to care

```
$ black --diff hmm.py
All done! ✨ 🍰 ✨
1 file left unchanged.

```

[isort](https://pypi.org/project/isort/) does though:

```
$ isort --diff hmm.py
--- /home/.../PycharmProjects/hmm_test/hmm.py:before       2018-10-30 07:42:37.413436
+++ /home/.../PycharmProjects/hmm_test/hmm.py:after        2018-10-30 09:08:01.170909
@@ -1,5 +1,5 @@
+import string
 from datetime import datetime
-import string
 
 d = datetime.now()
 s = string.ascii_letters

```

Reply:
It seems that most things either don't care, or want it after. We should probably go for after then to make things as easy as possible for anyone wanting to work on this.